style.css

```diff
@@ -634,8 +632,9 @@ textarea::placeholder {
     background-color: #3c3a39;
   }
 
-  .nav .fa-bars {
+  .hamburger {
     display: none;
+    cursor: pointer;
```

Comment:
- [ ] when hovering over a list item in the nav menu it enlarges the text however this causes the first part of the text to not be visible, this is not in line with the style guidelines, please make a change so the full text is visible also when hovering over the text.

![afbeelding](https://github.com/Mahdi-Niazi/Portfolio_Mobile_Setup/assets/39240820/8ccbcdcd-d989-4cb2-91d8-357f2f18d30e)